nautobot/docs/installation/tables/v2-api-behavior-changes.yaml

```diff
@@ -59,6 +59,9 @@
 - Model: "TenantGroup"
   Field: "tenant_count"
   Changes: "Now only counts Tenants directly belonging to this TenantGroup, not those belonging to its descendants"
+- Model: "VirtualChassis"
+  Field: "name"
+  Changes: "Now must be globally unique"
```

Comment:
This should be in `-database-behavior-changes` more than `-api-behavior-changes`, IMO.